tests/test_move.py

```diff
@@ -1,388 +1,304 @@
-                                       [mock_card, mock_card_other_colors, mock_card]]
-        info_dict = {'player_id': 1, 'information_type': 'color', 'information': 'blue'}
-        move = Move(move_type='give_information', player_index=0, information=info_dict)
-        mock_gamestate = move.apply(game_state=mock_gamestate)
-        mock_card.make_public.assert_called_with('color')
-        assert mock_card.make_public.call_count == 2
-        mock_gamestate.board.use_clock_token.assert_called_once()
-
-    def test_apply_give_information_number(self):
-        """Give information (number) should:
-            call make_public('number') on all cards with that number in a hand
-            call board.use_clock_token()
-        """
-        mock_gamestate = create_autospec(GameState)
-        mock_gamestate.board = create_autospec(Board)
-        mock_gamestate.board.clock_tokens = 1
-        mock_card = create_autospec(Card)
-        mock_card.number = 2
-        mock_card_other_number = create_autospec(Card)
-        mock_card_other_number.number = 4
-        mock_gamestate.player_hands = [[mock_card_other_number, mock_card, mock_card],
-                                       [mock_card, mock_card_other_number, mock_card_other_number]]
-        info_dict = {'player_id': 1, 'information_type': 'number', 'information': 2}
-        move = Move(move_type='give_information', player_index=0, information=info_dict)
-        mock_gamestate = move.apply(game_state=mock_gamestate)
-        mock_card.make_public.assert_called_once_with('number')
-        mock_gamestate.board.use_clock_token.assert_called_once()
-
-    def test_apply_give_information_number_multiple_cards(self):
-        """Give information (number) should:
-            call make_public('number') on all cards with that number in a hand
-            call board.use_clock_token()
-        """
-        mock_gamestate = create_autospec(GameState)
-        mock_gamestate.board = create_autospec(Board)
-        mock_gamestate.board.clock_tokens = 1
-        mock_card = create_autospec(Card)
-        mock_card.number = 2
-        mock_card_other_number = create_autospec(Card)
-        mock_card_other_number.number = 4
-        mock_gamestate.player_hands = [[mock_card_other_number, mock_card, mock_card_other_number],
-                                       [mock_card, mock_card_other_number, mock_card]]
-        info_dict = {'player_id': 1, 'information_type': 'number', 'information': 2}
-        move = Move(move_type='give_information', player_index=0, information=info_dict)
-        mock_gamestate = move.apply(game_state=mock_gamestate)
-        mock_card.make_public.assert_called_with('number')
-        assert mock_card.make_public.call_count == 2
-        mock_gamestate.board.use_clock_token.assert_called_once()
+import util
+
+
+# TODO: Are some of these integration tests?
+
+@pytest.fixture(scope='function')
+def card():
```

Comment:
Same as in board: for mocks, name the function with mock so it's explicit this fixture/input is a mock